src/assets/ebike_data.js

```diff
@@ -223,6 +223,165 @@ export const data = {
+      accessories: ['basket', 'fenders', 'front rack', 'head light', 'rear rack','aventon battery',],
+      price: 1499,
+      sizes: ['S/M', 'M/L'],
+      weight: 53,
+      weight_capacity_lbs: 300,
+      top_speed_mph: 28,
+      wheel_size_in: 27.5,
+      tire_width_in: 2.1,
+      colors: ['Polar', 'Himalayan'],
+      battery_capacity_ah: 14,
+      battery_voltage: 48,
+      controller_a: -1,
+      motor_location: 'hub drive',
+      motor_wattage: 750,
+      motor_brand: 'No Data',
+      sensor: 'torque',
+      torque_max_nm: -1,
+      range: 60,
+      brake_type: 'hydraulic',
+      speeds: 8,
+      image_url: ['https://cdn.shopify.com/s/files/1/1520/2468/products/level.2-polar-2.jpg?v=1663041589&width=1200', 'https://cdn.shopify.com/s/files/1/1520/2468/products/level.2-himalayan-2.jpg?v=1663041589&width=1200'],
+      website:'https://www.aventon.com/products/aventon-level-step-through-commuter-ebike',
+      throttle_control: 'thumb',
+      crank_set_teeth: 46
+    },
+    {
+      id: 11,
+      archive: false,
+      brand: 'Aventon',
+      model: 'Level Commuter',
+      class: 3,
+      style: ['thin tire', 'front suspension', 'urban',''],
+      accessories: ['basket', 'fenders', 'front rack', 'head light', 'rear rack',],
+      price: 1799,
+      sizes: ['S/M', 'M/L'],
+      weight: 62,
+      weight_capacity_lbs: 300,
+      top_speed_mph: 28,
+      wheel_size_in: 27.5,
+      tire_width_in: -1,
+      colors: ['stone gray'],
+      battery_capacity_ah: 14,
+      battery_voltage: 48,
+      controller_a: -1,
+      motor_location: 'hub drive',
+      motor_wattage: 750,
+      motor_brand: 'No Data',
+      sensor: 'cadence',
+      torque_max_nm: -1,
+      range: 40,
+      brake_type: 'hydraulic',
+      speeds: 8,
+      image_url: ['https://cdn.shopify.com/s/files/1/1520/2468/products/2020_Aventon_Level_StoneGrey_StudioPhotos_001.jpg?v=1649284315&width=1200'],
+      website:'https://www.aventon.com/products/level-ebike-v1',
+      throttle_control: 'thumb',
+      crank_set_teeth: 46
+    },
+    {
+      id: 12,
+      archive: false,
+      brand: 'Aventon',
+      model: 'Level Commuter',
+      class: 3,
+      style: ['thin tire', 'front suspension', 'urban',''],
+      accessories: ['basket', 'fenders', 'front rack', 'head light', 'rear rack',],
+      price: 1799,
+      sizes: ['S/M', 'M/L'],
+      weight: 62,
+      weight_capacity_lbs: 300,
+      top_speed_mph: 28,
+      wheel_size_in: 27.5,
+      tire_width_in: -1,
+      colors: ['stone gray'],
+      battery_capacity_ah: 14,
+      battery_voltage: 48,
+      controller_a: -1,
+      motor_location: 'hub drive',
+      motor_wattage: 750,
+      motor_brand: 'No Data',
+      sensor: 'cadence',
+      torque_max_nm: -1,
+      range: 40,
+      brake_type: 'hydraulic',
+      speeds: 8,
+      image_url: ['https://cdn.shopify.com/s/files/1/1520/2468/products/2020_Aventon_Level_StoneGrey_StudioPhotos_001.jpg?v=1649284315&width=1200'],
```

Comment:
Please add 2 more image URLs to this array

Reply:
there is only one color 

can u tell me what parts of the bike images to send